ResidentsSocialCarePlatformApi/V1/Controllers/SocialCareController.cs

```diff
@@ -60,5 +63,17 @@ public IActionResult ViewRecord(int mosaicId)
             }
         }
 
+        /// /// <summary>
+        /// Returns list of cases notes for a Person/Mosaic ID
+        /// </summary>
+        /// <response code="200">Success. Returns a list of matching case note information</response>
+        // TO DO: Error Handling & responses????
```

Comment:
👍 Might be worth updating the PR description's following up actions to have this.